Workout/Info.plist

```diff
@@ -18,8 +18,18 @@
 	<string>$(MARKETING_VERSION)</string>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>GADApplicationIdentifier</key>
+	<string>ca-app-pub-7085161342725707~3715618473</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSBluetoothPeripheralUsageDescription</key>
+	<string>Bluetooth may be used while interacting with ads.</string>
+	<key>NSCalendarsUsageDescription</key>
+	<string>Calendar data my be used while interacting with ads.</string>
+	<key>NSHealthShareUsageDescription</key>
+	<string>Access to data is used to package it in CSV files for exporting purposes.</string>
+	<key>NSHealthUpdateUsageDescription</key>
+	<string>Access to data is used to package it in CSV files for exporting purposes.</string>
```

Comment:
These 4 should also be localized using a `.strings` file: https://stackoverflow.com/a/25736915/5031210